doc/release-notes/release-7.sgml.in

```diff
@@ -104,6 +108,17 @@ This section gives an account of those changes in three categories:
 
 </descrip>
 
+<sect>Cache Manager changes since Squid-@SQUID_RELEASE_OLD@<label id="mgr">
```

Comment:
We do not have to add this new section, but I think it is a good idea because most Cache Manager changes are not tightly correlated to any configuration directives and are best described in a dedicated section, where admins would have a better chance of finding/noticing/contextualizing them. 

I do not insist on this addition.

Reply:
IMO this ```Cache Manager Changes``` is suitable for the section subtitle used in the ```Major Changes``` menu of new specific feature changes.
Changes to Cache Manager itself have far too many objections blocking updates. Which in turn make alterations to this area far too rare to deserve their own section.

Reply:
Sorry I named the section wrongly. To be clear the text describing this change should be added to the template like this:

```sgml
...
<itemize>
  <item>Cache Manager changes
</itemize
<p>Most user-facing changes are reflected in squid.conf (see below).

<sect1>Cache Manager changes
<p> More info in the <url url="https://wiki.squid-cache.org/Features/CacheManager/Index" description="Squid wiki" />.

<p>[ your description of the report/action removal ]

<sect>Changes to squid.conf since ....
```

Reply:
I still do not understand why we should move the description of these Cache Manager changes into the "Major new features..." section, but I did as requested (see commit 0562a0c). That commit uses the [change request](https://github.com/squid-cache/squid/pull/1378#discussion_r1232242400) template with some minor polishing touches (mainly SGML markup fixes).

FWIW, the resulting HTML file is temporary [available](https://gist.github.com/rousskov/7b838002468aafdea2f2d9bae3149578#file-releasenotes-7-html).

Reply:
Thank you. IMO this could do with more polish, but is good enough for now and we can do that later in the lifecycle when the feature changes are increased.